src/components/Member/index.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Button, Icon, Segment} from 'semantic-ui-react';
+import { TaskContainerWrapper, TaskItem, TaskInput } from '../../styles/CreateTaskForm';
+
+class Member extends React.Component {
+
+  handleRemoveActivity = (idx) => {
+    const { member, handleRemoveActivity, index } = this.props;
+    const activitiessAfterRemove = member.activities.filter((a, aidx) => idx !== aidx);
+    handleRemoveActivity(activitiessAfterRemove, index);
+  };
+
+  handleActivityChange = (idx) => (e) => {
+    const { member, handleActivityChange, index } = this.props;
+    const newTasks = member.activities.map((task, tidx) => {
+      if (idx !== tidx) return task;
+      return { ...task, [e.target.name]: e.target.value };
+    });
+
+    handleActivityChange(newTasks, index);
+  };
+
+  handleAddActivity = () => {
+    const { member, handleAddActivity, index} = this.props;
+    const newTask = { name: '', url: '', logtime: '' };
+    handleAddActivity(newTask, index);
+  };
+
+  render() {
+    const { member } = this.props;
+    const userActivities = this.props.member.activities;
+    return (
+      <Segment compact>
+        <p>{member.user.name}</p>
+        {userActivities.map((task, index) => (
+          // eslint-disable-next-line
+          <TaskContainerWrapper key={index}>
```

Comment:
I guess this `// eslint-disable-next-line` happens because `index` is used as key, 
it's time to use uuidv1 I guess

Reply:
We cannot because we deal with dynamic form